drizzlepac/hlautils/catalog_utils.py

```diff
@@ -65,16 +63,17 @@ def __init__(self, filename):
     def close(self):
         self.imghdu.close()
 
-    def build_kernel(self, fwhmpsf, scale):
+    def build_kernel(self, box_size, win_size, fwhmpsf, scale):
         if self.bkg is None:
-            self.compute_background()
+            self.compute_background(box_size, win_size)
 
-        self.kernel = astrometric_utils.build_auto_kernel(self.data, self.wht_image,
-                                                          threshold=self.bkg.background_rms, fwhm=fwhmpsf / scale)
+        self.kernel, self.kernel_fwhm = astrometric_utils.build_auto_kernel(self.data,
+                                                                            self.wht_image,
+                                                                            threshold=self.bkg_rms_ra,
+                                                                            fwhm=fwhmpsf / scale)
 
-    def compute_background(self, box_size=BKG_BOX_SIZE, win_size=BKG_FILTER_SIZE,
-                           bkg_estimator=SExtractorBackground, rms_estimator=StdBackgroundRMS,
-                           nsigma=5., threshold_flag=None):
+    def compute_background(self, box_size, win_size,
+                           bkg_estimator=SExtractorBackground, rms_estimator=StdBackgroundRMS):
```

Comment:
The variable 'bkg_dao_rms' is not used and can be removed from this method.

Reply:
Issue resolved in commit [c631ee9](https://github.com/spacetelescope/drizzlepac/pull/455/commits/c631ee978a71b8519922f8783c215cb65a6165a8)